### PR TITLE
add email reminders

### PR DESCRIPTION
**Pull request by IAlexHilimovich**
Sending email reminders when book period of keeping has expired

### CODE REVIEW

**Comment by ngame2011** [comment]
@Chichmarin17 @Yoinezero review this PR please

**Review by Yoinezero — APPROVED** [review]

File: packages/apiserver/src/scheduler/scheduler.service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { ReminderService } from '../reminder';
+
+@Injectable()
+export class SchedulerService {
+  constructor(private reminderService: ReminderService) {}
+
+  @Cron('0 * * * *')
```
Comment (Chichmarin17) [inline packages/apiserver/src/scheduler/scheduler.service.ts:9]: How often this method will be invoked acc to `0 * * * *` ? Each hour?

**Review by Chichmarin17 — APPROVED** [review]
Can not test how it works as I have no credentials to run the service, but looks fine

File: packages/apiserver/src/scheduler/scheduler.service.ts (hunk 1 above)
Comment (IAlexHilimovich) [reply packages/apiserver/src/scheduler/scheduler.service.ts:9]: yes

**Pull request merged**